linux/sound.cpp

```diff
@@ -108,7 +98,7 @@ void CSound::OpenJack ( const bool bNoAutoJackConnect, const char* jackClientNam
     // tell the JACK server that we are ready to roll
     if ( jack_activate ( pJackClient ) )
     {
-        throw CGenErr ( tr ( "Cannot activate the Jack client." ) );
+        throw CGenErr ( tr ( "Cannot activate the JACK client." ) );
```

Comment:
Does the "client" here mean Jamulus? Can we recommend an action to the user?

Reply:
No. As far as I understand it's basically the audio API connection which Jamulus uses. For the user, I doubt it makes a difference. Not even sure when this error appears.

Reply:
"Check the JACK output".

Reply:
I think this is resolved now.